### DISCUSSION TITLE
Seperate click counter for mouse clicks and keyboard taps

### DISCUSSION THREAD

**Discussion by showerosu** [ppy/osu#26007] [Ideas]
Mouse buttons aren't currently displayed in Lazer, which is concerning for playstyles like hybrid and mouse only and the only similar discussions are quite old, so wondering if there are any plans to fix/add this like it is in stable?

**Comment by bdach** [comment]
Doesn't seem horribly important to bring this back for me. We support a whole host of input devices, not just keyboard and mouse, including midi, tablet pen, controller etc. Bringing back the notion of keyboard/mouse into the key counter will needlessly complicate this.

**Reply by showerosu** [reply]
I don't think it is wrong to say that mouse buttons are the second most used form of tapping, not to mention a lot of mouse only and hybrid players would feel very poorly about it never being added again as there will be nothing to discern them from the rest of players. Perhaps a mouse only mod or something along those lines but that seems more complicated than just being the same as stable, not that doing this seems complicated in the first place.
